lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,181 @@
+                                        "name": "credit_provider",
+                                        "value": "edx"
+                                    }
+                                ],
+                                "stockrecords": [
+                                    {
+                                        "price_currency": "USD",
+                                        "product": 123,
+                                        "partner_sku": "1234ABC",
+                                        "partner": 1,
+                                        "price_excl_tax": "10.00",
+                                        "id": 123
+                                    }
+                                ],
+                                "product_class": "Seat",
+                                "title": "Dummy title",
+                                "url": "https://ecom.edx.org/api/v2/products/123/",
+                                "price": "10.00",
+                                "expires": null,
+                                "is_available_to_buy": true,
+                                "id": 123,
+                                "structure": "child"
+                            },
+                            "line_price_excl_tax": "10.00",
+                            "description": "dummy description",
+                            "title": "dummy title",
+                            "quantity": 1
+                        }
+                    ],
+                    "number": "EDX-123456",
+                    "date_placed": "2016-01-01T01:01:01Z",
+                    "currency": "USD",
+                    "total_excl_tax": "10.00"
+                };
+                providerResponseData = {
+                        "id": "edx",
+                        "display_name": "edX",
+                        "url": "http://www.edx.org",
+                        "status_url": "http://www.edx.org/status",
+                        "description": "Nothing",
+                        "enable_integration": false,
+                        "fulfillment_instructions": "",
+                        "thumbnail_url": "http://edx.org/thumbnail.png"
+                };
+                courseResponseData = {
+                    "id": "course-v1:edx+dummy+2015_T3",
+                    "name": "receipt test",
+                    "category": "course",
+                    "org": "edx",
+                    "run": "2015_T2",
+                    "course": "CS420",
+                    "uri": "http://test.com/api/course_structure/v0/courses/course-v1:edx+dummy+2015_T3/",
+                    "image_url": "/test.jpg",
+                    "start": "2030-01-01T00:00:00Z",
+                    "end": null
+                };
+
+            });
+
+            var createReceiptView = function() {
+                return new ReceiptView({el: $('#receipt-container')});
+            };
+
+            var loadReceiptFixture = function(){
```

Comment:
Add space between `()` and `{`, like `var loadReceiptFixture = function() {`
Also add this var at top of this method.